WordPress/Classes/ViewRelated/Comments/CommentTableHeaderView.swift

```diff
@@ -0,0 +1,156 @@
+            switch self {
+            case .post:
+                return Constants.postCommentSubText
+            case .reply(let authorName):
+                return String(format: Constants.replyCommentSubTextFormat, authorName)
+            case .commentThread:
+                return Constants.commentThreadSubText
+            }
+        }
+    }
+
+    private let hostingController: UIHostingController<CommentHeaderView>
+
+    init(title: String,
+         subtitle: Subtitle,
+         showsDisclosureIndicator: Bool = false,
+         reuseIdentifier: String? = CommentTableHeaderView.defaultReuseID) {
+        let headerView = CommentHeaderView(title: title,
+                                           subtitle: subtitle,
+                                           showsDisclosureIndicator: showsDisclosureIndicator)
+        hostingController = .init(rootView: headerView)
+        super.init(reuseIdentifier: reuseIdentifier)
+        configureView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - Private methods
+
+private extension CommentTableHeaderView {
+
+    func configureView() {
+        hostingController.view.translatesAutoresizingMaskIntoConstraints = false
+        hostingController.view.backgroundColor = .clear
+        contentView.addSubview(hostingController.view)
+        contentView.pinSubviewToAllEdges(hostingController.view)
+    }
+
+    enum Constants {
+        static let postCommentSubText = NSLocalizedString(
+            "comment.header.subText.post",
+            value: "Comment on",
+            comment: """
+                Provides a hint that the current screen displays a comment on a post.
+                The title of the post will be displayed below this text.
+                Example: Comment on \n My First Post
+                """
+        )
+
+        static let replyCommentSubTextFormat = NSLocalizedString(
+            "comment.header.subText.reply",
+            value: "Reply to %1$@",
+            comment: """
+                Provides a hint that the current screen displays a reply to a comment.
+                %1$@ is a placeholder for the comment author's name that's been replied to.
+                Example: Reply to Pamela Nguyen
+                """
+        )
+
+        static let commentThreadSubText = NSLocalizedString(
+            "comment.header.subText.commentThread",
+            value: "Comments on",
+            comment: """
+                Sentence fragment.
+                The full phrase is 'Comments on' followed by the title of the post on a separate line.
+                """
+        )
+    }
+}
+
+// MARK: - SwiftUI
+
+private struct CommentHeaderView: View {
+
+    @State var title = String()
+    @State var subtitle: CommentTableHeaderView.Subtitle = .post
+    @State var showsDisclosureIndicator = true
+
+    var body: some View {
+        if #available(iOS 15.0, *) {
```

Comment:
You are hopefully going to be able to remove this (and `VisualEffectView`) soon. We're discussing when to merge the [deployment target bump](https://github.com/wordpress-mobile/WordPress-iOS/pull/20917).

Reply:
Thanks for the heads up, and looking forward to it! I'll create a follow-up PR to remove them once the version bump PR gets merged.